solara/components/applayout.py

```diff
@@ -257,7 +257,19 @@ def AppLayout(
         use_drawer = True
     title = t.use_title_get() or title
     children_appbartitle = apptitle_portal.use_portal()
-    show_app_bar = (title and (len(routes) > 1 and navigation)) or children_appbar or use_drawer or children_appbartitle
+    v_slots = []
+    # Separate tabs from the other children. We want to keep a reference to any tabs that were added through children
+    # so we can make sure AppBar is rendered even when tabs are the only children
+    tabs = None
+    for child_appbar in children_appbar.copy():
+        if child_appbar.component == solara.lab.Tabs:
+            if tabs is not None:
+                raise ValueError("Only one Tabs component is allowed in the AppBar")
+            tabs = child_appbar
+            children_appbar.remove(tabs)
+    tabs_to_render = tabs
```

Comment:
I don't understand the need for this line, why do we need an alias?
I do think btw that `tabs` is a bad name (the plural makes it feel like it should be a list). Assuming we can get rid of this line, maybe we can call tabs, tabs_element (to make it clear it is 1 thing)